azerothcore/azerothcore-wotlk
# feat(Core/GameEvents)Add new command & load function to reload game event npc vendors

## Changes Proposed:
This PR proposes changes to:

## Issues Addressed:
- Closes #20306 

## SOURCE:
The changes have been validated through:

## Tests Performed:
This PR has been:


## How to Test the Changes:
1. .event start 10 (Childrens week)
2. .npc add 29478
3. add a new entry in the game_event_npc_vendor database for guid 99369 with item 100 
4. reload the table & see the new item in the vendor list
5. .event stop 10, to see all items of the event being removed as you´d expect.

## Known Issues and TODO List:


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
